tests/linen/module_test.py

```diff
@@ -768,18 +768,6 @@ def bar(self):
         # attributes defined during `setup`
         self.assertEqual(empty.bar(), 3)
 
-    def test_call_unbound_noncompact_module_methods(self):
```

Comment:
Why as this test removed?

Reply:
It was duplicated. See how it looked before https://github.com/google/flax/blob/3448764de94461bd08caef2cf90e113a711ffa3c/tests/linen/module_test.py#L759

Reply:
It was duplicated. See how it looked before https://github.com/google/flax/blob/3448764de94461bd08caef2cf90e113a711ffa3c/tests/linen/module_test.py#L759-L769 and https://github.com/google/flax/blob/3448764de94461bd08caef2cf90e113a711ffa3c/tests/linen/module_test.py#L771-L781